lib/frontend/get-jira-configuration.js

```diff
@@ -42,6 +42,7 @@ module.exports = async (req, res, next) => {
         isGlobalInstall: data.repository_selection === 'all',
         installedAt: formatDate(data.updated_at),
         syncState: data.syncState,
+        maintenanceMode: process.env.MAINTENANCE_MODE === 'true' || false,
```

Comment:
Do you need this?  Maintenance mode is global and not per installation, right?

Reply:
nope, just removed it and used `../maintenanceMode` in the template ... that seems to access to top-level field.

Reply:
yep, correct :)